test/extended/images/mysql_replica.go

```diff
@@ -0,0 +1,143 @@
+package images
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util/rand"
+)
+
+var _ = g.Describe("images: MySQL replication template", func() {
+	defer g.GinkgoRecover()
+	var (
+		persistentVolumes []*kapi.PersistentVolume
+		templatePath      = "https://raw.githubusercontent.com/openshift/mysql/master/5.5/examples/replica/mysql_replica.json"
+		oc                = exutil.NewCLI("mysql-replication", exutil.KubeConfigPath())
+		user              = "user"
+		userPass          = "userpass"
+		rootPass          = "rootpass"
+		database          = "userdb"
+	)
+
+	g.Describe("MySQL replication", func() {
+		g.JustBeforeEach(func() {
+			var err error
+			persistentVolumes, err = exutil.SetupHostPathVolumes(oc.AdminKubeREST().PersistentVolumes(), "mysql-replication", "512Mi", 1)
+			o.Expect(err).NotTo(o.HaveOccurred())
+		})
+
+		g.AfterEach(func() {
```

Comment:
It's ok for now, though I'm going to change it because I'll be adding another template.